Packages/Newtonsoft.Json-for-Unity.Converters/Configuration/ConverterConfig.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Newtonsoft.Json.UnityConverters.Configuration
+{
+#pragma warning disable CA2235 // Mark all non-serializable fields
+    [Serializable]
+    [System.Diagnostics.DebuggerDisplay("{converterName}, enabled={enabled}")]
+    public struct ConverterConfig
+    {
+        public bool enabled;
+
+        public string converterName;
+
+        public List<KeyedConfig> settings;
+
+        public ConverterConfig(Type converter, IEnumerable<KeyedConfig> settings = null)
+        {
+            enabled = true;
+            converterName = converter?.AssemblyQualifiedName;
+            this.settings = settings?.ToList() ?? new List<KeyedConfig>();
+        }
+
+        public ConverterConfig(string converterName, IEnumerable<KeyedConfig> settings = null)
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)